netconf/rpc.go

```diff
@@ -140,8 +140,8 @@ func MethodGet(filterType string, dataXml string) RawMethod {
 }
 
 // MethodEditConfig files a NETCONF edit-config request with the remote host
-func MethodEditConfig(database string, dataXml string) RawMethod {
-	return RawMethod(fmt.Sprintf(editConfigXml, database, dataXml))
+func MethodEditConfig(database string, errorMethod string, dataXml string) RawMethod {
```

Comment:
This breaks the API for others.  We would need a new function to not be breaking.  However i like the idea.